ironfish/src/wallet/account.ts

```diff
@@ -548,7 +395,8 @@ export class Account {
             'nullifierToNote mappings must have a corresponding decryptedNote',
           )
 
-          await this.updateDecryptedNote(
+          await this.walletDb.saveDecryptedNote(
```

Comment:
So I understand this correctly. The only real logic change here is that we are no longer attempting to delete the nullifier for these notes. And that is because spends will always have nullifiers so no reason to ever delete them?

Reply:
Yeah exactly, the spends will still have nullifiers, and those would be removed only if the transaction is disconnected from the chain